tests/lax_numpy_test.py

```diff
@@ -1789,6 +1802,23 @@ def testArgsortManually(self):
     expected = onp.argsort(x)
     self.assertAllClose(expected, ans, check_dtypes=False)
 
+  def testMsortManually(self):
+    x = onp.array([16, 15, 23, 42, 8, 4])
+    ans = lnp.msort(x)
+    expected = onp.msort(x)
+    self.assertAllClose(expected, ans, check_dtypes=False)
```

Comment:
I'd suggest adding a call to `CompileAndCheck`, which verifies that a function produces the same result with and without `jit`. (If you try to do so for the other two functions, you will find that it fails, for the reasons I gave above. Try it!)

Reply:
I added `CompileAndCheck` and `CheckAgainstNumpy `.
I tried thank you!